lib/widgets-page.php

```diff
@@ -125,3 +125,21 @@ function gutenberg_widgets_init( $hook ) {
 	wp_enqueue_style( 'wp-format-library' );
 }
 add_action( 'admin_enqueue_scripts', 'gutenberg_widgets_init' );
+
+/**
+ * Tells the script loader to load the scripts and styles of custom block on widgets editor screen.
+ *
+ * @param bool $is_block_editor_screen Current decision about loading block assets.
+ * @return bool Filtered decision about loading block assets.
+ */
+function widgets_editor_load_block_editor_scripts_and_styles( $is_block_editor_screen ) {
```

Comment:
Convention says this should start with `gutenberg_` (e.g. `gutenberg_should_load_block_editor_scripts_and_styles`) because it's a function added by the Gutenberg plugin.